description: fix undefined behavior, variables assigned twice in the same statement
author: Michael Gilbert <mgilbert@debian.org>

--- a/cc/layers/surface_layer_impl.cc
+++ b/cc/layers/surface_layer_impl.cc
@@ -118,7 +118,7 @@ viz::SurfaceDrawQuad* SurfaceLayerImpl::
   // allocate a new SharedQuadState. Assign the new SharedQuadState to
   // *|common_shared_quad_state| so that it may be reused by another emitted
   // viz::SurfaceDrawQuad.
-  viz::SharedQuadState* shared_quad_state =
+  viz::SharedQuadState*
     shared_quad_state = render_pass->CreateAndAppendSharedQuadState();
 
   PopulateScaledSharedQuadState(shared_quad_state, device_scale_factor,
--- a/chrome/browser/ui/libgtkui/settings_provider_gsettings.cc
+++ b/chrome/browser/ui/libgtkui/settings_provider_gsettings.cc
@@ -43,8 +43,8 @@ SettingsProviderGSettings::SettingsProvi
   const gchar* settings_schema =
       base::nix::GetDesktopEnvironment(env.get()) ==
               base::nix::DESKTOP_ENVIRONMENT_CINNAMON
-          ? settings_schema = kCinnamonPreferencesSchema
-          : settings_schema = kGnomePreferencesSchema;
+          ? kCinnamonPreferencesSchema
+          : kGnomePreferencesSchema;
 
   if (!g_settings_schema_source_lookup(g_settings_schema_source_get_default(),
                                        settings_schema, FALSE) ||
